python/cugraph/cugraph/dask/link_analysis/pagerank.py

```diff
@@ -105,8 +157,20 @@ def pagerank(input_graph,
         numerical roundoff. Usually values between 0.01 and 0.00001 are
         acceptable.
 
-    nstart : not supported
-        initial guess for pagerank
+    nstart : cudf.Dataframe, optional (default=None)
+        GPU Dataframe containing the initial guess for pagerank.
+        nstart['vertex'] : cudf.Series
+            Subset of vertices of graph for initial guess for pagerank values
+        nstart['values'] : cudf.Series
+            Pagerank values for vertices
+
+    weight: str, optional (default=None)
+        The attribute column to be used as edge weights if Graph is a NetworkX
+        Graph. This parameter is here for NetworkX compatibility and is ignored
+        in case of a cugraph.Graph
```

Comment:
Right. I should probably remove that parameter because it was never supported in previous releases. I copied the `docstrings` from the SG implementation and I forgot to clean this